data/datasets/stackexchange/stats.md

```diff
@@ -0,0 +1,186 @@
+- 3dprinting: 1,006
```

Comment:
These statistics are duplicated (here and in `README.md`), I think it could make sense to just have them in one place to make easier to keep up to date in case the dataset or conversion process changes in future

Reply:
Agreed, I'll just leave it in the README.